### DISCUSSION TITLE
Is it possible to adjust graphics config files?

### DISCUSSION THREAD

**Discussion by professional-cynic** [netbrain/zwift#162] [Q&A] [answered]
I would like to change the default graphic settings: C:\Program Files (x86)\Zwift\data\configs as described [here](https://zwiftinsider.com/config-file-tweaks/). Is it possible?

**Comment by netbrain** [comment]
Yes, it is possible to change the default graphic settings in Zwift.

However, inside the container, Zwift’s configuration files, including graphics settings, are stored in a virtualized Windows file system, which is mounted in Docker. To adjust the graphic settings, you would need to:

1. Access the Docker container: You can get a shell inside the running Zwift container by using:

docker exec -it <container_name> /bin/bash

Replace <container_name> with the actual name of the Zwift container (e.g., zwift-$USER).


2. Navigate to the Zwift configuration folder: Inside the container, Zwift's files, including the graphic settings, might be under a Wine emulated path. This is typically something like:

/home/user/.wine/drive_c/Program Files (x86)/Zwift/data/configs

Here, you should be able to locate and modify the prefs.xml or other configuration files that control graphics settings.

You might be able to mount you own copy of prefs.xml into the container with the -v flag.

**Reply by professional-cynic** [reply]
Yes, but I'm using podman and for whatever reason podman doesn't see the Zwift container even if I use podman container ls -a.

**Reply by professional-cynic** [reply]
Apparently, it works only when the Zwift container is running.

**Reply by netbrain** [reply]
Yes there's a volume attached, see if you can find the files there

**Reply by professional-cynic** [reply]
So, when I accessed the container while Zwift was running, I couldn't find anything in Program Files in Wine, and I don't see anything related in the attached volume.

**Reply by professional-cynic** [reply]
I've found it: ~/.local/share/containers/storage/overlay/cda9b9ff969d0f3f202837a70a0e0465164d5d8fb608262e7935b7d9ca10bdd6/diff1/home/user/.wine/drive_c/Program%20Files%20(x86)/Zwift/data/configs

**Reply by rommeswi** [reply]
What is the default setting in there? I'd like to set it to the lowest possible to smoothen the fps.

**Reply by glennvl** [reply, marked as answer]
It is now possible to use `ZWIFT_OVERRIDE_GRAPHICS` and `ZWIFT_OVERRIDE_RESOLUTION` to customize the zwift graphics. See the `How can I access/modify the graphics settings?` section in the [README](https://github.com/netbrain/zwift/blob/master/README.md) file for details on how to use them.
